tests/e2e/lib/plan-helper.js

```diff
@@ -314,6 +314,67 @@ function getPlan( type ) {
 			},
 		};
 	}
+
+	if ( type === 'jetpack_free' ) {
```

Comment:
We have something very similar here: https://github.com/Automattic/jetpack/blob/658661b2d80afc2f5d203c5c42fc5d2d55c30bb4/tests/php/general/test_class.jetpack-plan.php#L117

Should we try to share it?

Reply:
good point - have just added a todo note about it at this stage, but if this is not approved before I get the Instagram stuff out of the way will try and take a look at it in this PR.